service/src/main/resources/application.yml

```diff
@@ -20,6 +20,19 @@ logging:
   level.bio.terra.externalcreds: ${EXTERNALCREDS_LOG_LEVEL:debug}
   pattern.level: '%X{requestId} %5p'
 
+otel:
+  sdk:
+    disabled: false # set to true to disable all open telemetry features
+  traces:
+    sampler:
+      probability: ${SAMPLING_PROBABILITY:0}
```

Comment:
this was incorrect in earlier iterations, sampling is set [like this](https://github.com/DataBiosphere/terra-workspace-manager/blob/3958b8cb975444950305a74a126d59fcc9974136/service/src/main/resources/application.yml#L182)